src/components/Launch/index.js

```diff
@@ -29,38 +29,53 @@ export class Launch extends React.Component {
         const data = this.props.data;
         return (
             <section className="launch" >
-                <LaunchImage links={data.links} imgBool={this.state.imgBool} />
+                <Route path={"/" + data.mission_name}
```

Comment:
String interpolation!

I need to take a closer look at the router - this looks a bit different than I remember. We can look at this together in person.